peripherals/dma/dma2d_common.yaml

```diff
@@ -0,0 +1,85 @@
+# "Chrom-Art Accelerator" 2D DMA engine
+# Common fields
+
+"DMA2D*":
+  CR:
+    MODE:
+      MemoryToMemory: [0, "Memory-to-memor (FG fetch only)"]
+      MemoryToMemoryPFC: [1, "Memory-to-memory with PFC (FG fetch only with FG PFC active)"]
+      MemoryToMemoryPFCBlending: [2, "Memory-to-memory with blending (FG and BG fetch with PFC and blending)"]
+      RegisterToMemory: [3, "Register-to-memory"]
+    CEIE:
+      Disable: [0, "CE interrupt disable"]
+      Enable: [1, "CE interrupt enable"]
+    CTCIE:
+      Disable: [0, "CTC interrupt disable"]
+      Enable: [1, "CTC interrupt enable"]
+    CAEIE:
+      Disable: [0, "CAE interrupt disable"]
+      Enable: [1, "CAE interrupt enable"]
+    TWIE:
+      Disable: [0, "TW interrupt disable"]
+      Enable: [1, "TW interrupt enable"]
+    TCIE:
+      Disable: [0, "TC interrupt disable"]
+      Enable: [1, "TC interrupt enable"]
+    TEIE:
+      Disable: [0, "TE interrupt disable"]
+      Enable: [1, "TE interrupt enable"]
+    ABORT:
+      AbortRequest: [1, "Transfer abort requested"]
+    SUSP:
+      NotSuspended: [0, "Transfer not suspended"]
+      Suspended: [1, "Transfer suspended"]
+    START:
+      Start: [1, "Launch the DMA2D"]
+  IFCR:
+    CCEIF:
+      Clear: [1, "Clear the CEIF flag in the ISR register"]
+    CCTCIF:
+      Clear: [1, "Clear the CTCIF flag in the ISR register"]
+    CAECIF:
+      Clear: [1, "Clear the CAEIF flag in the ISR register"]
+    CTWIF:
+      Clear: [1, "Clear the TWIF flag in the ISR register"]
+    CTCIF:
+      Clear: [1, "Clear the TCIF flag in the ISR register"]
+    CTEIF:
+      Clear: [1, "Clear the TEIF flag in the ISR register"]
+  "[FB]GMAR":
+    MA: [0, 4294967295]
```

Comment:
I agree this would be a valid (and maybe also good?) use of unsafe. But I added the annotation for consistence with the existing DMA descriptions (eg. [dma_v3](https://github.com/stm32-rs/stm32-rs/blob/22a6fabfdad01ef6527259baf4f0d731111d60d1/peripherals/dma/dma_v3.yaml#L88)).

I've now removed them.

Reply:
Thanks! I guess we should consider removing them from the other DMA peripherals, really...